src/azul/health.py

```diff
@@ -93,6 +94,25 @@ def queues(self):
                 }
         return response
 
+    @memoized_property
+    def failures(self):
+        failed_bundle_notifications = []
+        database = boto3.resource('dynamodb')
+        table = database.Table(config.dynamo_failure_message_table_name)
+        bundle_key_condition = {'MessageType': {'AttributeValueList': ['bundle_notification'],
+                                                'ComparisonOperator': 'EQ'}}
+        bundle_notification_items = table.query(KeyConditions=bundle_key_condition, Limit=100)['Items']
+        for item in bundle_notification_items:
+            body = json.loads(item['Body'])
+            failed_bundle_notifications.append({'bundle_uuid': body['notification']['match']['bundle_uuid'],
```

Comment:
This is too much knowledge about the notification format in a place where it doesn't belong. I think we should just include the entire message verbatim in the response.